src/Elastic.Markdown/BuildContext.cs

```diff
@@ -72,8 +72,7 @@ public BuildContext(DiagnosticsCollector collector, IFileSystem readFileSystem,
 		if (ConfigurationPath.FullName != DocumentationSourceDirectory.FullName)
 			DocumentationSourceDirectory = ConfigurationPath.Directory!;
 
-
-		Git = GitCheckoutInformation.Create(DocumentationSourceDirectory, ReadFileSystem);
+		Git = GitCheckoutInformation.Create(rootFolder, ReadFileSystem);
```

Comment:
This was `SourceDirectory` because assembler cannot rely on `rootFolder` 

There is some traversing logic on line 50 in `GitCheckoutInformation.cs` to attempt to get the relative .git folder so not sure if this is needed. 

Long term `CheckoutDirectory` needs to be explicitly set in `BuildContext` vs relying on `Paths.Root`. (not needed as part of this PR).

Can you validate `DocumentationSourceDirectory` indeed does not work with the traversal logic? 

e.g in docs-content the docs are at the root of `DocumentationSourceDirectory` so is the `.git` folder. For most other's it actually lives on folder up from `docs`.

Reply:
See also: https://github.com/elastic/docs-builder/pull/791 which takes away this ambiguity.